tools/integration-tests/tester/tests/common/common_test.go

```diff
@@ -27,36 +31,36 @@ func TestSynchronization(t *testing.T) {
 
 	// 1. issue data messages
 	ids := tests.SendDataMessagesOnRandomPeer(t, n.Peers(), numMessages)
-
-	// wait for messages to be gossiped
-	time.Sleep(10 * time.Second)
+	log.Printf("Issuing %d messages to be synced... done\n", numMessages)
+	// wait for messages to be gossiped and leaf the synced time windows
+	time.Sleep(framework.ParaTangleTimeWindow)
```

Comment:
It has to sleep for `ParaTangleTimeWindow` as otherwise the node will not go out of sync.
But we can sure just decrease the value of `ParaTangleTimeWindow` in the integration tests, I think it even used to be 30s at some point

Reply:
Ah! I changed it to 2 min for debugging :stuck_out_tongue: